week-1-ts-ecosystem/w01e03-object-oriented-structure/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    "target": "ES2020",
```

Comment:
Nie jestem pewien, ale być może problem z `#` w prywatnych polach zniknie, jak użyjesz `esnext` 🤔

Reply:
nope, dalej to samo